Add limit_tokens plugin (#58)
digitalocean/do-markdownit: README.md

Prism.highlightAll();
```

## limit_tokens

<details>
<summary>Filters and transforms tokens in the token stream.</summary>

**Options:**

Pass options for this plugin as the `limit_tokens` property of the `do-markdownit` plugin options.
Set this property to `false` to disable this plugin.

> - `allowedTokens` (`array`): A list Markdown tokens that should render

Reviewer: ```suggestion
- `allowedTokens` (`array`): A list of Markdown tokens that should render
```

Author: Updated.

- `transformTokens` (`object`): An object where the keys are Markdown tokens that should be transformed.

> The transformation is done base on the value which is an `object` with two keys: `type` and `htmlTag`.

Reviewer: ```suggestion
The transformation is done based on the value which is an `object` with two keys: `type` and `htmlTag`.
```

</details>

```js
const Prism = require('@digitalocean/do-markdownit/vendor/prismjs');

require('@digitalocean/do-markdownit/vendor/prismjs/plugins/toolbar/prism-toolbar')(Prism);
require('@digitalocean/do-markdownit/vendor/prismjs/plugins/copy-to-clipboard/prism-copy-to-clipboard')(Prism);

Prism.highlightAll();

> ```

Reviewer 2: This seems like the wrong place to put this, as the Prism Keep HTML plugin is now a subheading of this.

Author: Can not count to 3... I did not see the difference between the two headings...
Moved the limit_tokens after all the Prism stuff

> ```

Reviewer 2: This seems rather unrelated?

Author: Looks like an accidental extra duplication.

Prism.highlight('console.log("<mark>Hello, world!</mark>");', Prism.languages.javascript, 'javascript');
```

## limit_tokens

<details>
<summary>Filters and transforms tokens in the token stream.</summary>

**Options:**

Pass options for this plugin as the `limit_tokens` property of the `do-markdownit` plugin options.
Set this property to `false` to disable this plugin.

> - `allowedTokens` (`array`): A list of Markdown tokens that should render

Author: Changed.

> - `transformTokens` (`object`): An object where the keys are Markdown tokens that should be transformed.

Author: Changed.

>   The transformation is done based on the value which is an `function` that expects a Markdown token as a param

Reviewer 2: ```suggestion
  The transformation is done based on the value which is a `function` that expects a [Markdown token](https://markdown-it.github.io/markdown-it/#Token) as a param
```
Probably worth including a link back to the docs here

Author: Updated.

>   The transformation is done based on the value which is a `function` that expects a [Markdown token](https://markdown-it.github.io/markdown-it/#Token) as a param

Author: Removed.
